assets/js/atomic/blocks/product-elements/summary/supports.ts

```diff
@@ -6,7 +6,7 @@ import { isFeaturePluginBuild } from '@woocommerce/block-settings';
 export const supports = {
 	...( isFeaturePluginBuild() && {
 		color: {
-			text: true,
+			text: '',
```

Comment:
Looking at the type definition of `BlockSupports`:

```
    /**
     * This property adds block controls which allow the user
     * to set text color in a block.
     *
     * @defaultValue true
     */
    text: string;
```

and the Supports API:

```
[color](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-supports/#color)
Type: Object
Default value: null
Subproperties:
background: type boolean, default value true
__experimentalDuotone: type string, default value undefined
gradients: type boolean, default value false
link: type boolean, default value false
text: type boolean, default value true
```
I think the type definition of `BlockSuports` is incorrect and should be fixed.

In the meantime, we can fix the type error by removing the `text` property as text color support is enabled by default.

Reply:
Thanks for mentioning this. Besides removing `text: true`, I also removed `link: false` as this sub property is also using the default value.